client/state/themes/selectors/get-theme-signup-url.js

```diff
@@ -14,7 +15,11 @@ export function getThemeSignupUrl( state, themeId ) {
 		return null;
 	}
 
-	let url = '/start/with-theme?ref=calypshowcase&theme=' + themeId;
+	const flow = ! doesThemeBundleSoftwareSet( state, themeId )
```

Comment:
Very minor nit but this might be a little easier to read as

```
const flow = doesThemeBundleSoftwareSet( state, themeId )

  ? 'with-business-theme'
  : 'with-theme';
```